### DISCUSSION TITLE
modal popup on node click not working

### DISCUSSION THREAD

**Discussion by amir-alleyne** [xyflow/xyflow#3221] [Q&A]
<EmptyDiv className="reactflow-wrapper" css={{
        width:'100%',
        height:'100%',
        pointerEvents: 'all',

    }}>
    <ReactFlow
      nodes={nodes}
      edges={edges}
      onNodesChange={onNodesChange}
      onEdgesChange={onEdgesChange}
      onConnect={onConnect}
      elementsSelectable={true}
      onNodeClick={(event, node) => {Modal.setVisible(true)}}
      nodeTypes={nodeTypes}
      panOnDrag={false}
      nodesFocusable={true}
       zoomOnScroll={true}
    //     zoomOnDoubleClick={false}
        nodesDraggable={false}
    fitView={true}
    >
      <Background />
    </ReactFlow>
    </EmptyDiv>

I have this reactflow diagram, but when I click the nodes the modal does not appear. Strangely enough when I click the nodes using the dev tool cursor the modal pops up. It seems to be only working for mobile onClick events but not website onClick events

**Comment by amir-alleyne** [comment]
Solved! The problem was not defining custom node types as global but inside the component.

**Reply by freddifederica** [reply]
Hi @amir-alleyne, I am having a similar issue. Could you please help me figure out how to get the popup to show up? Is this Modal a react flow component that you created yourself? Or did you import it from another library? Thank you very much!
